modules/extension/xsd/xsd-gml3/src/test/java/org/geotools/gml3/bindings/MeasureTypeBindingTest.java

```diff
@@ -34,7 +33,7 @@ public void testParser() throws Exception {
         assertNotNull(measure);
         assertEquals(1234, measure.doubleValue(), 0.1);
 
-        assertEquals("http://someuri", ((BaseUnit) measure.getUnit()).getSymbol());
+        assertEquals("http://someuri", measure.getUnit().getSymbol());
```

Comment:
Why did the prior code require a cast to BaseUnit? Has the api changed to make getSymbol() visible?

Reply:
It was also visible in Units of Measure 1.0 as well. Not sure maybe in JSR-275 it wasn't.